torchinductor/codecache.py

```diff
@@ -68,6 +68,18 @@ def cpp_compiler_search(search):
             continue
     raise exc.InvalidCxxCompiler()
 
+@functools.lru_cache(1)
+def cpp_supports_flag(flag: str) -> bool:
+   rc = subprocess.run([cpp_compiler(), "-E", "-x", "c", flag, "-"], stdout=subprocess.PIPE, stderr=subprocess.PIPE, input="")
+   return rc.returncode == 0
+
+
+
+def cxx_flags() -> str:
+    flags = "-march=native -O3 -ffast-math -fno-finite-math-only"
```

Comment:
Can you make this config driven, but on by default? I feel like there is value in looking at the unoptimized code, as -O3 can inline and remove a lot of stuff.